tests/test_search.py

```diff
@@ -467,67 +469,60 @@ def test_get_version_path_prefix_from_obj(
-    _search("display", search_root=meshing.tui)
-    lines = capsys.readouterr().out.splitlines()
-    assert "<search_root>.display (Object)" in lines
-    _search("display", search_root=meshing.tui.display)
-    lines = capsys.readouterr().out.splitlines()
-    assert "<search_root>.update_scene.display (Command)" in lines
-    assert "<search_root>.display_states (Object)" in lines
-    _search("cad", search_root=meshing.meshing)
-    lines = capsys.readouterr().out.splitlines()
-    assert "<search_root>.GlobalSettings.EnableCleanCAD (Parameter)" in lines
-    assert "<search_root>.LoadCADGeometry (Command)" in lines
-    _search("next", search_root=meshing.workflow)
-    lines = capsys.readouterr().out.splitlines()
-    assert '<search_root>.TaskObject["<name>"].InsertNextTask (Command)' in lines
-    _search("next", search_root=meshing.workflow.TaskObject)
-    lines = capsys.readouterr().out.splitlines()
-    assert '<search_root>["<name>"].InsertNextTask (Command)' in lines
-    _search("next", search_root=meshing.workflow.TaskObject["Import Geometry"])
-    lines = capsys.readouterr().out.splitlines()
-    assert "<search_root>.InsertNextTask (Command)" in lines
-    _search("timeout", search_root=meshing.preferences)
-    lines = capsys.readouterr().out.splitlines()
-    assert "<search_root>.General.IdleTimeout (Parameter)" in lines
-    _search("timeout", search_root=meshing.preferences.General)
-    lines = capsys.readouterr().out.splitlines()
-    assert "<search_root>.IdleTimeout (Parameter)" in lines
-
-
+    results = _search("display", search_root=meshing)
+    assert "<search_root>.tui.display (Object)" in results
+    results = _search("display", search_root=meshing.tui)
+    assert "<search_root>.display (Object)" in results
+    results = _search("display", search_root=meshing.tui.display)
+    assert "<search_root>.update_scene.display (Command)" in results
+    assert "<search_root>.display_states (Object)" in results
+    results = _search("cad", search_root=meshing.meshing)
+    assert "<search_root>.GlobalSettings.EnableCleanCAD (Parameter)" in results
+    assert "<search_root>.LoadCADGeometry (Command)" in results
+    results = _search("next", search_root=meshing.workflow)
+    assert '<search_root>.TaskObject["<name>"].InsertNextTask (Command)' in results
+    results = _search("next", search_root=meshing.workflow.TaskObject)
+    assert '<search_root>["<name>"].InsertNextTask (Command)' in results
+    results = _search(
+        "next", search_root=meshing.workflow.TaskObject["Import Geometry"]
+    )
+    assert "<search_root>.InsertNextTask (Command)" in results
+    results = _search("timeout", search_root=meshing.preferences)
+    assert "<search_root>.General.IdleTimeout (Parameter)" in results
+    results = _search("timeout", search_root=meshing.preferences.General)
+    assert "<search_root>.IdleTimeout (Parameter)" in results
+
+
+@pytest.mark.skip("Results are varying each time.")
```

Comment:
@mkundu1 We are skipping this test as results are varying each time.

https://github.com/ansys/pyfluent/actions/runs/9480112747/job/26122446207?pr=2901

Reply:
We can run those just with a single Fluent version, I'll look into it after merging the current PR.

Reply:
Ok. As per our last discussion, we have made all other test version 24.2 as a stable single Fluent version and I thought 23.2 is also a stable one therefore I have not updated it. Thank you.

Reply:
Yeah, but the results can differ between 24.2 and 23.2. These tests gives the required code coverage while running with a single Fluent version, so we don't need to run them with multiple Fluent versions.

Reply:
Yes, got it. Thank you.